monkey/agent_plugins/credentials_collectors/chrome/src/linux_database_reader.py

```diff
@@ -0,0 +1,41 @@
+import logging
+import os
+import shutil
+import sqlite3
+from collections.abc import Callable, Iterator
+from pathlib import Path
+
+logger = logging.getLogger(__name__)
+
+DB_TEMP_PATH = "/tmp/chrome.db"
```

Comment:
Don't use static or predictable files paths in `/tmp`, it can lead to security issues. Use `mkstemp()` or `NamedTemporaryFile`.